skygear-0.5.0.slide

```diff
@@ -0,0 +1,53 @@
+Skygear.io at Oursky
+0.5.0 Release
+19 Feb 2016
+
+Rick Mak
+Oursky
+rickmak@oursky.com
+@rickmak
+CC81327D
+
+* Important Changes
+
+- zmq is now depreacted
+- `http` and `exec` will remove c deps
+
+You can now directly install by
+
+    go install github.com/oursky/skygear
```

Comment:
Not sure if you want to mention that skygear can be downloaded from https://github.com/oursky/skygear/releases